deepmd/entrypoints/test.py

```diff
@@ -254,6 +254,83 @@ def save_txt_file(
         np.savetxt(fp, data, header=header)
 
 
+def ener_err(
```

Comment:
What do you mean by the name `ener_err` and `ener_err_ops`??
They are not good names and can share almost code.